conduct/faq.md

```diff
@@ -0,0 +1,77 @@
+# Project Jupyter Code of Conduct - Frequently Asked Questions
+
+This FAQ attempts to address common questions and concerns around the Jupyter
+community's Code of Conduct. If you still have questions after reading it,
+please feel free to contact us at
+[*conduct@jupyter.org*](mailto:conduct@jupyter.org).
+
+### Why have you adopted a Code of Conduct?
+
+We think the Jupyter community is awesome. If you're familiar with the Jupyter
+community, you'll probably notice that the Code basically matches what we
+already do. Think of this as documentation: we're taking implicit expectations
+about behavior and making them explicit.
+
+We're doing this because the Jupyter community is growing faster than any of us
+could have anticipated. This is on balance a very positive thing, but as we've
+grown past the point where it's possible to know the whole community, we think
+it's very important to be clear about our values.
+
+We know that the Jupyter community is open, friendly, and welcoming. We want to
+make sure everyone else knows it too.
+
+### What does it mean to "adopt" a Code of Conduct?
+
+For the most part, we don't think it means large changes. We think that the text
+does a really good job describing the way the Jupyter community already conducts
+itself. We expect that most people will simply continue to behave as they have
+in the past.
+
+However, we do expect that people will abide by the spirit and words of the Code
+of Conduct when in "official" Jupyter spaces. This code has been adopted by both
+the Jupyter Steering Council and JupyterDay/JupyterCon events. That means that
+it will apply both in community spaces and at our events.
+
+In practice, this means mailing lists, the various Jupyter channels, and Jupyter
```

Comment:
> True, it would be repetitive but likely grammatically better to have a concrete noun instead of "this".

Especially since this is a new paragraph.